### PR TITLE
V3: Allows tiles to be closed without focusing on tile

### PR DESCRIPTION
**Pull request by eireland**
Allows closing of tile on first click instead of having to focus on tile first.

### CODE REVIEW

File: v3/src/components/calculator/calculator-title-bar.tsx [hunk 1]
```diff
@@ -17,7 +17,7 @@ export const CalculatorTitleBar = observer(function CalculatorTitleBar({tile, on
       <Flex className="header-right">
         <MinimizeIcon className="component-minimize-icon" title={t("DG.Component.minimizeComponent.toolTip")}/>
         <CloseButton className="component-close-button" title={t("DG.Component.closeComponent.toolTip")}
-          onClick={()=>onCloseTile(tileId)}/>
+          onPointerDownCapture={()=>onCloseTile(tileId)}  onClick={()=>onCloseTile(tileId)}/>
```
Comment (kswenson) [inline v3/src/components/calculator/calculator-title-bar.tsx:20]: Presumably you don't need the `onClick` handler if you've got the `onPointerDownCapture` handler. Out of curiosity, did you try an `onPointerDown` handler?

**Review by kswenson — CHANGES_REQUESTED** [review]
👍 Looks good -- should remove the `onClick` handlers assuming they're no longer necessary. Also try `onPointerDown` instead of `onPointerDownCapture` if you haven't already.

**Review by kswenson — APPROVED** [review]
👍 LGTM

**Review by bfinzer — APPROVED** [review]
👍 LGTM

**Pull request merged**